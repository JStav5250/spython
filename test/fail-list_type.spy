float l = [12.4, 37.6, 46.5, 1]

print(2) 
