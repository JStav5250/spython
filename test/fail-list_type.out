TypeError: data type must be the same t: int type: float!!
